src/components/settings/SettingsLMSTab/ExistingLMSCardDeck.jsx

```diff
@@ -39,6 +39,7 @@ const ExistingLMSCardDeck = ({
   // const listItems = timeSort(configData).map((config) => (
   const listActive = configData.filter(config => getStatus(config) === 'active').map(config => (
     <ExistingCard
+      key={`${config.channelCode}${config.id}`}
```

Comment:
lack of keys was causing exceptions to clutter up web console, so I cleaned up the file.